content/docs/introduction/faq.mdx

```diff
@@ -170,9 +181,9 @@ Generally this happens when there is branch protection on your main branch and T
 
 To fix this issue, either:
 
-* Upgrade from legacy branch protection to branch rulesets - [https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-rulesets/creating-rulesets-for-a-repository](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-rulesets/creating-rulesets-for-a-repository)\
+- Upgrade from legacy branch protection to branch rulesets - [https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-rulesets/creating-rulesets-for-a-repository](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-rulesets/creating-rulesets-for-a-repository)\
   Ensure that the TinaCloud app is allowed to bypass the rule requiring branch protection
-* Enable [Editorial Workflow](/editorial-workflow) to create PRs on the branch
+- Enable [Editorial Workflow](/editorial-workflow) to create PRs on the branch
 
 ### How do I resolve `Invalid or undefined branch` errors?
```

Comment:
Hi @bradystroud

- [ ] Change from

> How do I resolve `Invalid or undefined branch` errors?

to

> How to resolve `Invalid or undefined branch` errors?

Reply:
Disagree - the suggestion is not a question